fleet-management/src/main/java/cz/fi/muni/pa165/entity/Employee.java

```diff
@@ -43,4 +43,29 @@ public void setName(String name) {
     public void setSurname(String surname) {
         this.surname = surname;
     }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
+
+        Employee employee = (Employee) o;
+
+        return id != null ? id.equals(employee.id) : employee.id == null;
```

Comment:
This equals isn't correct, we cant use only id in it. I will refactor this today.